modules/fbs-core/api/src/main/resources/migrations/06_task_requirement_type.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN;
+
+alter table task
+    add requirement_type VARCHAR(200) not null;
```

Comment:
* do we really need 200 characters here?
* a default value should be set to have already existing Task return the correct `requirementType` (`DEFAULT 'mandatory'`)